src/test/java/api/requests/RequestsAPICreationTests.java

```diff
@@ -543,6 +543,76 @@ void cannotCreateRequestWithNonExistentRequestLevelWhenTlrEnabled() {
       hasParameter("requestLevel", "invalid"))));
   }
 
+  @Test
+  void canCreateTlrRequestWhenUserHasNoLoanForAnyItemsOfInstance() {
+    reconfigureTlrFeature(TlrFeatureStatus.ENABLED);
+
+    IndividualResource instanceMultipleCopies = instancesFixture.basedUponDunkirk();
+    IndividualResource holdings = holdingsFixture.defaultWithHoldings(
+      instanceMultipleCopies.getId());
+    IndividualResource locationsResource = locationsFixture.mainFloor();
+
+    IndividualResource individualResource = itemsFixture.basedUponDunkirkWithCustomHoldingAndLocation(holdings.getId(),
+      locationsResource.getId());
+
+    String instanceId = holdings.getJson().getString("instanceId");
+    String holdingsId = holdings.getJson().getString("id");
+
+    final UUID pickupServicePointId = servicePointsFixture.cd1().getId();
+
+    checkOutFixture.checkOutByBarcode(individualResource, usersFixture.steve());
+
+    final IndividualResource response = requestsClient.create(
+      new RequestBuilder()
+        .hold()
+        .withPickupServicePointId(pickupServicePointId)
+        .titleRequestLevel()
+        .forItem(individualResource)
+        .withInstanceId(UUID.fromString(instanceId))
+        .withHoldingsRecordId(UUID.fromString(holdingsId))
+        .by(usersFixture.jessica())
+        .create());
+
+    assertThat(response.getJson().getInteger("position"), CoreMatchers.is(1));
+  }
+
+  @Test
+  void cannotCreateTlrRequestWhenUserHasLoanForSomeItemsOfInstance() {
+    reconfigureTlrFeature(TlrFeatureStatus.ENABLED);
+
+    IndividualResource instanceMultipleCopies = instancesFixture.basedUponDunkirk();
+    IndividualResource holdings = holdingsFixture.defaultWithHoldings(
+      instanceMultipleCopies.getId());
+    IndividualResource locationsResource = locationsFixture.mainFloor();
+
+    IndividualResource individualResource = itemsFixture.createItemWithHoldingsAndLocation(holdings.getId(),
+      locationsResource.getId());
+    IndividualResource individualResource2 = itemsFixture.createItemWithHoldingsAndLocation(holdings.getId(),
+      locationsResource.getId());
+
+    JsonObject holdingsJson = holdings.getJson();
+
+    final UUID pickupServicePointId = servicePointsFixture.cd1().getId();
+
+    checkOutFixture.checkOutByBarcode(individualResource2, usersFixture.jessica());
+
+    Response postResponse = requestsClient.attemptCreate(
```

Comment:
I would suggest decreasing the number of empty lines and using them only when we really need to separate some blocks of code by logical meaning. It will significantly reduce the number of lines

Reply:
done